core/src/main/res/values/strings.xml

```diff
@@ -323,4 +323,6 @@
   <string name="open_article">Open Article</string>
   <string name="delete_note_dialog_message">Note: Notes are not deleted from your storage</string>
   <string name="delete_selected_notes">Delete Selected Notes?</string>
+  <string name="download_tts_language_title">Download this language for read aloud feature</string>
```

Comment:
Maybe something User-friendly :)

Reply:
Not sure to have myself something better…